controller/handler/handler.go

```diff
@@ -30,6 +30,7 @@ func CreateRouter(staticFolder string, securityConfig *security.Config, uiConfig
 	unprotected.Handle("/v1/config", ConfigHandler{securityConfig: securityConfig}).Methods("GET")
 	protected.HandleFunc("/v1/endpoints/statuses", EndpointStatuses).Methods("GET") // No GzipHandler for this one, because we cache the content as Gzipped already
 	protected.HandleFunc("/v1/endpoints/{key}/statuses", GzipHandlerFunc(EndpointStatus)).Methods("GET")
+	unprotected.HandleFunc("/v1/endpoints/{key}/status/badge.svg", StatusBadge).Methods("GET")
```

Comment:
I'm a little unsure about that path.. It's not consistent with the other badges.
Maybe it should be `/v1/endpoints/{key}/current-status/badge.svg` instead? 
I was thinking `/v1/endpoints/{key}/statuses/latest/badge.svg`, but it doesn't really make sense.

At this point I'm just thinking out loud, but I think `/v1/endpoints/{key}/health/badge.svg` makes more sense, but the badge would need to be renamed to the `health` badge, so `StatusBadge` would need to be renamed to `HealthBadge` & same with the other references to it being a `status badge` instead of a `health badge`.

What do you think?

Reply:
I think `current-status` or `status/current` could work for a status badge, but a `HealthBadge` with `/health/badge.svg` path makes more sense. I can make the changes tomorrow.

Reply:
Yeah let's go for health then :) Thanks!